spec/features/admin/products_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+describe 'Admin - Products', js: true do
+
+  context 'as Admin' do
+
+    xit 'should be able to change supplier' do
+      s1 = create(:supplier)
+      s2 = create(:supplier)
+      product = create :product
+      product.add_supplier! s1
+
+      login_user create(:admin_user)
+      visit spree.admin_product_path(product)
+
+      select2 s2.name, from: 'Supplier'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.